apps/mempool/config.json

```diff
@@ -0,0 +1,62 @@
+{
+   "$schema": "../schema.json",
+   "name": "Mempool",
+   "available": true,
+   "exposable": false,
+   "no_gui": false,
+   "port": 2200,
+   "id": "mempool",
+   "description": "Bitcoin mempool visualizer and blockchain explorer",
+   "tipi_version": 1,
+   "version": "v3.0.0-dev2",
```

Comment:
all versions should be stable (but can be overrided by user-config)

Reply:
The latest stable version is v2.5.0, can use that image version instead (same for the other one)

Reply:
Yes please

Reply:
Im using this version because the 2.5.0 does not have suport for the .cookie auth method

Reply:
That's fair enough for me, @JigSawFr would you be OK with keeping the existing image?

Reply:
@JigSawFr  @MontejoJorge Do you think we could get this PR unstuck? Last I checked there was some question about if it was OK to use the non stable image. Can we come to a conclusion on that?

Reply:
I cant decide about that, @JigSawFr @meienberger @steveiliop56 Any thoughts about this?

Reply:
If you wait long enough the Mempool guys will release 3.0.0 , they just tagged [`beta1`](https://github.com/mempool/mempool/releases/tag/v3.0.0-beta1) 😁

Reply:
@DireMunchkin [https://github.com/mempool/mempool/releases/tag/v3.0.0-rc1](rc1) is out, so comming soon i guess

Reply:
It's out now: https://github.com/mempool/mempool/releases/tag/v3.0.0